### ISSUE TITLE
what is it unreduced_pinball vs tfa.losses.PinballLoss

### ISSUE THREAD

**Issue by Sandy4321** [microsoft/knossos-ksc#1068]
what is it unreduced_pinball

    check_same_as_tensorflow(
        partial(losses.unreduced_pinball, tau=tau), tfa.losses.PinballLoss(tau=tau)
    )

from 
https://github.com/microsoft/knossos-ksc/blob/09b95f9df85305973101b72eb466c532e0b9438f/rlo/test/rlo/test_losses.py

**Comment by sarahnlewis** [comment]
`unreduced_pinball` is a pytorch implementation of pinball loss. The function is [here](https://github.com/microsoft/knossos-ksc/blob/09b95f9df85305973101b72eb466c532e0b9438f/rlo/src/rlo/losses.py). This test is left over from when we ported code from tensorflow to pytorch, and wanted to check we had equivalent implementations of the loss functions.
